traffic_ops/traffic_ops_golang/api/shared_handlers_test.go

```diff
@@ -142,7 +142,7 @@ func TestCreateHandler(t *testing.T) {
 	createFunc(w, r)
 
 	//verifies the body is in the expected format
-	body := `{"alerts":[{"text":"tester was created.","level":"success"}],"response":{"ID":1}}`
+	body := "{\"alerts\":[{\"text\":\"tester was created.\",\"level\":\"success\"}],\"response\":{\"ID\":1}}\n"
```

Comment:
This might be easier to read if you keep the backticks and instead just add `+"\n"` to the end. Does that work?

Same for the two lines below as well.

Reply:
sure